rapidsai/devel-rocky.arm64.Dockerfile

```diff
@@ -1,6 +1,6 @@
-ARG FROM_IMAGE=gpuci/miniforge-cuda-arm64
-ARG CUDA_VER=11.2
-ARG LINUX_VER=centos8
+ARG FROM_IMAGE=nvidia/cuda
```

Comment:
is there a reason the `FROM` image was changed here? We need these images to inherit from the `miniforge-cuda` images.

Reply:
I don't think the `FROM` image should change here. We build `gpuci/miniforge-cuda-arm64` [here](https://github.com/rapidsai/gpuci-build-environment/blob/branch-22.08/miniforge-cuda/Dockerfile).